release.sh

```diff
@@ -171,13 +171,18 @@ npm cache clean --force
 rm -rf node_modules/
 
 echo "==> Docker"
-docker build --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui:${VERSION} .
+docker build -f docker/Dockerfile-production --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui:${VERSION} .
+docker build -f docker/Dockerfile-production-no-analytics --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui-no-analytics:${VERSION} .
```

Comment:
I think we should declare a version with no analytics, not another docker

`--tag gisaia/arlas-wui:${VERSION}-no-analytics`

Reply:
fixed